src/Microsoft.Extensions.CommandLineUtils/project.json

```diff
@@ -10,7 +10,8 @@
     "nowarn": [
       "CS1591"
     ],
-    "xmlDoc": true
+    "xmlDoc": true,
+    "compile": "../Microsoft.Extensions.CommandLineUtils.Sources/**/*.cs"
```

Comment:
Curious: Why do we have two separate command line utils one as sources other as a package but both have the same content?